const e=JSON.parse('{"key":"v-315150fe","path":"/python/08.python-basis-exception.html","title":"异常处理","lang":"zh-CN","frontmatter":{"title":"异常处理","category":["Python"],"order":8},"headers":[{"level":2,"title":"1. 语法错误","slug":"_1-语法错误","link":"#_1-语法错误","children":[]},{"level":2,"title":"2. 异常的处理","slug":"_2-异常的处理","link":"#_2-异常的处理","children":[]},{"level":2,"title":"3. Python 主要的内置异常及其处理","slug":"_3-python-主要的内置异常及其处理","link":"#_3-python-主要的内置异常及其处理","children":[]},{"level":2,"title":"4. 手动抛出异常","slug":"_4-手动抛出异常","link":"#_4-手动抛出异常","children":[{"level":3,"title":"4.1. raise 语句","slug":"_4-1-raise-语句","link":"#_4-1-raise-语句","children":[]},{"level":3,"title":"4.2. assert 语句","slug":"_4-2-assert-语句","link":"#_4-2-assert-语句","children":[]}]},{"level":2,"title":"5. 自定义异常类","slug":"_5-自定义异常类","link":"#_5-自定义异常类","children":[]}],"git":{"createdTime":1676279113000,"updatedTime":1676279113000,"contributors":[{"name":"Luo","email":"943215317@qq.com","commits":1}]},"readingTime":{"minutes":6.17,"words":1851},"filePathRelative":"python/08.python-basis-exception.md","localizedDate":"2023年2月13日"}');export{e as data};
